Orange/data/table.py

```diff
@@ -1,27 +1,22 @@
+import operator
 import os
-import re
 import zlib
-from collections import MutableSequence, Iterable, Sequence, Sized
-from itertools import chain
-from numbers import Real, Integral
-import operator
+from collections import Iterable, Sequence, Sized
 from functools import reduce
-from warnings import warn
 from threading import Lock
-from tempfile import NamedTemporaryFile
-from urllib.parse import urlparse, unquote as urlunquote
-from urllib.request import urlopen
-from urllib.error import URLError
+from warnings import warn
+from numbers import Number
 
 import bottlechest as bn
 from scipy import sparse as sp
+from pandas import DataFrame, SparseDataFrame, Series, SparseSeries, Panel, SparsePanel, concat
```

Comment:
It's easy to see what belongs to upstream pandas. Like we import numpy as np, bottleneck as bn, scipy as sp, ..., and like is documented in [countless examples](http://pandas.pydata.org/pandas-docs/stable/dsintro.html). It's just a sane preference.